src/notsotuf/utils/file_handler.py

```diff
@@ -1,12 +1,25 @@
 import logging
+import os
 import pathlib
+import shutil
 import subprocess
 import sys
 from tempfile import NamedTemporaryFile
 from typing import Union
+import platform
 
 logger = logging.getLogger(__name__)
 
+def handle_update(
```

Comment:
In relation to issue #5, I'm going to rename the `move_and_exit` argument to `install`, so I would call this e.g. `install_update`. The present module I would like to call `platform_specific.py`. We can then do:

    from notsotuf.utils.platform_specific import install_update